### PR TITLE
Compliance to awesome repository requirements

### PR DESCRIPTION
**Pull request by glours**
Add
 - CODEOWNERS
 - MAINTAINERS
 - CONTRIBUTING.md

Fix `README.md` to be compliant with [awesome repository](https://github.com/sindresorhus/awesome#readme)

Signed-off-by: Guillaume Lours <guillaume.lours@docker.com>

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
+# Contributing
+
+Contributions should be made via pull requests. Pull requests will be reviewed
+by one or more maintainers and merged when acceptable.
+
+The goal of the Awesome Compose is to provide a curated list of application 
+samples that can be easily deployed with [Docker Compose](https://github.com/docker/compose).
+
+## Missing a kind of application? 
+You can request a new example of application by submitting an issue to our GitHub Repository.
```
Comment (chris-crone) [inline CONTRIBUTING.md]: nit: missing `\n` after heading

Comment (chris-crone) [reply CONTRIBUTING.md]:
```suggestion

You can request a new example of an application by submitting an issue to our GitHub repository.  
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+Contributions should be made via pull requests. Pull requests will be reviewed
+by one or more maintainers and merged when acceptable.
+
+The goal of the Awesome Compose is to provide a curated list of application 
+samples that can be easily deployed with [Docker Compose](https://github.com/docker/compose).
+
+## Missing a kind of application? 
+You can request a new example of application by submitting an issue to our GitHub Repository.  
+If you would like to implement a new application sample, please submit an issue with a proposal first,
```
Comment (chris-crone) [inline CONTRIBUTING.md]:
```suggestion
If you would like to submit a new application example, please start by submitting a proposal as an issue.
```

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,118 @@
...
+to be sure that we can use it.   
+Then we'll consult the community to know what can be the most valuable application with this technology/language/framework...
```
Comment (chris-crone) [inline CONTRIBUTING.md]:
```suggestion
The maintainers will then use this issue to discuss what the the most valuable example for the application, technology, language, or framework would be.
```

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,118 @@
...
+
+After the choice made, you can submit your pull request following the choice of the community without forgetting to add :
```
Comment (chris-crone) [inline CONTRIBUTING.md]:
```suggestion
After the choice has been made, you can submit your a pull request not forgetting to:
```

File: CONTRIBUTING.md [hunk 5]
```diff
@@ -0,0 +1,118 @@
...
+- a local README.md to describe the application and explain how to run/use the sample.
```
Comment (chris-crone) [inline CONTRIBUTING.md]:
```suggestion
- include an example README.md to describe the application and explain how to run/use the sample
```

File: CONTRIBUTING.md [hunk 6]
```diff
@@ -0,0 +1,118 @@
+# Contributing
+
+Contributions should be made via pull requests. Pull requests will be reviewed
+by one or more maintainers and merged when acceptable.
+
+The goal of the Awesome Compose is to provide a curated list of application 
+samples that can be easily deployed with [Docker Compose](https://github.com/docker/compose).
+
+## Missing a kind of application?
```
Comment (chris-crone) [inline CONTRIBUTING.md]:
```suggestion
## Missing an example?
```

File: CONTRIBUTING.md [hunk 7]
```diff
@@ -0,0 +1,118 @@
...
+You can request a new example of application by submitting an issue to our GitHub Repository.  
+If you would like to implement a new application sample, please submit an issue with a proposal first,
+to be sure that we can use it.   
+Then we'll consult the community to know what can be the most valuable application with this technology/language/framework...
+
+After the choice made, you can submit your pull request following the choice of the community without forgetting to add :
+- a local README.md to describe the application and explain how to run/use the sample.
+- edit the global README.md to add your sample in the repository main list.
+
+## Update a existing sample
```
Comment (chris-crone) [inline CONTRIBUTING.md]: We can actually remove this

File: CONTRIBUTING.md [hunk 8]
```diff
@@ -0,0 +1,118 @@
...
+
+Before submitting a new application, check if there isn't already application sample matching your need.
```
Comment (chris-crone) [inline CONTRIBUTING.md]: We should move this (and the next line) up to the other section

File: CONTRIBUTING.md [hunk 9]
```diff
@@ -0,0 +1,118 @@
...
+problem. If there are multiple possible solutions to the problem, these can be
+outlined in the issue. Once consensus is reached on how to resolve the issue, a
+pull request can be created.
+
+Pull requests that propose minor changes or improvements may be submitted
+without an associated issue or discussion.
+
+For large or high impact changes, contributors can reach out to maintainers
+before starting work. This will ensure that contributors and maintainers are
+aligned and increase the chance that the change is accepted.
```
Comment (chris-crone) [inline CONTRIBUTING.md]: I think you can remove this as it's more targeted at code/specs

**Review by chris-crone — CHANGES_REQUESTED** [review]
Couple of minor changes, thanks @glours!

File: CONTRIBUTING.md [hunk 10]
```diff
@@ -0,0 +1,103 @@
...
+The goal of the Awesome Compose is to provide a curated list of application 
+samples that can be easily deployed with [Docker Compose](https://github.com/docker/compose).
+
+## Missing an example? 
+
+You can request a new example of an application by submitting an issue to our GitHub repository. 
+
+Before submitting a new application, check if there isn't already application sample matching your need.
+
+If there is one, consider to update it instead of create a new one.
```
Comment (chris-crone) [inline CONTRIBUTING.md]: nit:
```suggestion
If there is one, consider updating it instead of creating a new one.
```

File: CONTRIBUTING.md [hunk 11]
```diff
@@ -0,0 +1,103 @@
...
+ 
+If you would like to submit a new application example, please start by submitting a proposal as an issue.
+The maintainers will then use this issue to discuss what the the most valuable example for the application,
+ technology, language, or framework would be.
+
+After the choice has been made, you can submit your a pull request not forgetting to:
```
Comment (chris-crone) [inline CONTRIBUTING.md]:
```suggestion
After the choice has been made, you can submit a pull request with the example remembering to:
```

**Review by chris-crone — CHANGES_REQUESTED** [review]
Minor changes after seeing it all together. Thanks @glours!

**Review by chris-crone — APPROVED** [review]
LGTM

**Review by mandervrik — COMMENTED** [review]
N

**Pull request merged**